pages/partials/attachFilePanel.ejs

```diff
@@ -0,0 +1,99 @@
+<div class="card mb-4" id="attach-file-panel">
+  <div class="card-header bg-secondary text-white">Attached files</div>
+  <table class="table table-sm">
+    <tbody>
+    <% if (file_list.length == 0) { %>
+    <tr><td><i>No attached files</i></td></tr>
+    <% } else { %>
+      <% file_list.forEach(function(file, iFile) { %>
+      <tr>
+        <td style="word-break:break-all;">
+          <i class="far fa-file-alt"></i>
+          <a class="attached-file" target="_blank" href="<%= urlPrefix %>/assessment_instance/<%= assessment_instance.id %>/file/<%= file.id %>/<%= file.display_filename %>">
+            <%= file.display_filename %>
+          </a>
+        </td>
+        <% if (assessment_instance.open) { %>
+        <td style="width:1%; text-align:right;">
+          <% if (file.type == 'student_upload') { %>
+          <a id="attachFileDeleteButton<%= iFile %>"
+             tabindex="0" class="btn btn-xs btn-secondary attachFileDeleteButton" role="button"
+             data-toggle="popover" data-container="body"
+             data-html="true" data-placement="auto" title="Confirm delete"
+             data-content="<%= include('attachFileDeleteForm',
```

Comment:
Nit: this is less important for smaller lists like this one, but I'd still prefer that we use the popover style used by the instructor gradebook popovers (attach data to button with `data-*` attributes, create and launch popover in code). Not blocking merge.

Reply:
Legit. I will skip for now.